tratto/src/main/java/star/tratto/util/FileUtils.java

```diff
@@ -0,0 +1,199 @@
+
+    /**
+     * The method creates a file {@link File} within a given directory.
+     *
+     * @param dirPath the path to the directory where the file must be saved
+     * @param fileName the name of the file where to write the content
+     * @param fileFormat the format of the file {@link FileFormat}.
+     *
+     * @return the file created {@link File}
+     * @throws IOException If the file cannot be created
+     */
+    public static File createFile(String dirPath, String fileName, FileFormat fileFormat) throws FolderCreationFailedException, IOException, FileNotCreatedException {
+        String filePath = Paths.get(dirPath, fileName + fileFormat.getValue()).toString();
+        File dir = new File(dirPath);
+        File file = new File(filePath);
+        // create directory.
+        if (!dir.exists()) {
+            boolean success = dir.mkdirs();
+            if (!success) {
+                String errMsg = String.format("Unable to create folder %s", dirPath);
+                logger.error(errMsg);
+                throw new FolderCreationFailedException();
+            }
+        }
+        // create file.
+        if (!file.exists()) {
+            boolean fileCreated = file.createNewFile();
+            if (!fileCreated) {
+                String errMsg = String.format("Unable to create file %s to path %s.", fileName, dirPath);
+                logger.error(errMsg);
+                throw new FileNotCreatedException();
+            }
+        }
+        return file;
+    }
+
+    /**
+     * The method gets the list of all the files that exist within a given directory.
+     *
+     * @param dir the directory from which to extract all the files contained
+     * @return the list of all the files existing within the given directory
+     */
+    public static List<File> extractJavaFilesFromDirectory(File dir) {
+        List<File> javaFileList = new ArrayList<>();
+        for (File file: Objects.requireNonNull(dir.listFiles())) {
+            if (file.isDirectory()) {
+                javaFileList.addAll(extractJavaFilesFromDirectory(file));
+            }
+            if (isJavaFile(file)) {
+                javaFileList.add(file);
+            }
+        }
+        return javaFileList;
+    }
+
+    /**
+     * The method builds a complete path to a file.
+     *
+     * @param dirPath the path to the directory where the file must be saved
+     * @param fileName the name of the file where to write the content
+     * @param fileFormat the format of the file {@link FileFormat}
+     * @return the complete path to a file.
+     */
+    public static String getAbsolutePathToFile(String dirPath, FileName fileName, FileFormat fileFormat) {
+        return Paths.get(dirPath, fileName.getValue()) + fileFormat.getValue();
+    }
+
+    /**
+     * The method checks if a file is a Java file.
+     * @param file the file to inspect
+     * @return a boolean value: {@code true} if the file is a Java file, {@code false} otherwise.
```

Comment:
In a Javadoc `@param`, `@return`, etc. clause, the initial text is a sentence fragment that starts with a lowercase (not capital) letter and does not end with a period unless followed by another sentence.